README.md

```diff
@@ -0,0 +1,81 @@
+# Konbata Converter
+
+Konbata converts .doc and .docx files into Canvas .imscc packages.
```

Comment:
"Konbata converts _and uploads_ .doc and .docx files _that have the specific naming structure (as described below)_ into Canvas .imscc packages." Or something like that that, at a glance, states that this is a specialized tool.